NOTICE.txt

```diff
@@ -1,7 +1,7 @@
 Elastic APM Server
 Copyright 2014-2021 Elasticsearch BV
 
-This product includes software developed by The Apache Software
+This product includes software developed by The Apache Software 
```

Comment:
whitespaces are back :)

Reply:
I dunno what's up, I just ran `make update`. I'll look closer if it happens again :shrug: